CHANGELOG.md

```diff
@@ -4,6 +4,68 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.4](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.4) - 2022-01-26
```

Comment:
Mergify will merge this PR as soon as there is an approval, so I'm leaving this comment to stop that happening until Janito is ready to tag.

Reply:
I'm ready I believe. I mean, we can get it merged and I can tag it tomorrow, right?